pyblish_lite/util.py

```diff
@@ -67,3 +67,18 @@ def u_print(msg, **kwargs):
             pass
         msg = msg.encode(encoding or 'utf-8', 'replace')
     print(msg, **kwargs)
+
+
+def collect_families_from_instances(instances, only_active=False):
```

Comment:
Please move single-use functions into the module where it is used.

Reply:
Used in model.py and control.py